source/documentation/other-topics/apiversion-changes-cronjob-k8s-1-25.html.md.erb

```diff
@@ -1,33 +1,43 @@
 ---
 title: Updating CronJob API version for Cloud Platform - deprecated Kubernetes v1.21
```

Comment:
deprecated Kubernetes v1.25

Reply:
I have updated the title to include removed in 1.25